server/admin.go

```diff
@@ -34,6 +34,7 @@ import (
 const (
 	LivenessPath    = "/live"
 	MetricsPath     = "/metrics"
+	ConfigPath      = "/config"
```

Comment:
This is better if it's under `/emulator` as that contains more admin endpoints for the emulator. So the route would be `/emulator/config`

Reply:
Moved :)